pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, cairo
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  # litemdview -v
+  version = "0.0.32";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "g0tsu";
+    repo = "litemdview";
+    rev = "eacbbc837e";
+    hash = "sha256-sD0y+/7TogywrvftxEXT1EyBf3BXH9QWbawFVOF8xRc=";
+  };
+
+  buildInputs = [
+    gtkmm3
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    pkg-config
+    libtool
+    cairo
+  ];
+
+  buildPhase = "
+    ./bootstrap
+    ./configure --prefix=$out
+    make
```

Comment:
You should be able to omit the `buildPhase`, if `make` is the sole command to be run.